templates/Web/Projects/VueDefault/_postaction.gitignore

```diff
@@ -1,12 +1,10 @@
-.DS_Store
-node_modules
-/build
-
 # vscode configuration
 .vscode/
 
-# Hide API Keys
-.env
+//{[{
+.DS_Store
+
+/build
 
 # production
 /server/build
```

Comment:
Should this be here? seems like this is duplicated in .gitignore when I generate a project with a backend

Reply:
Good catch! This should not be here.